plugin-server/src/main/services/http-server.ts

```diff
@@ -1,38 +1,47 @@
+import * as Sentry from '@sentry/node'
 import { createServer, IncomingMessage, Server, ServerResponse } from 'http'
 
-import { healthcheck } from '../../healthcheck'
```

Comment:
is this still used by something? Can we nuke it?

Reply:
Possibly